snc.sh

```diff
@@ -33,6 +33,7 @@ SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_ecdsa
 SCP="scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_ecdsa_crc"
 MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp}
 CERT_ROTATION=${SNC_DISABLE_CERT_ROTATION:-enabled}
+USE_PATCHED_KAO_KCMO_IMAGE=${SNC_USE_PATCHED_KAO_KCMO_IMAGE:-disabled}
```

Comment:
Or just `USE_PATCHED_RELEASE_IMAGE`? with a comment explaining that this release image uses patched KAO/KCMO with 1 year certificates?